packages/react/src/ReactElement.js

```diff
@@ -221,11 +221,34 @@ export function jsx(type, config, maybeKey) {
   // <div {...props} key="Hi" />, because we aren't currently able to tell if
   // key is explicitly declared to be undefined or not.
   if (maybeKey !== undefined) {
-    key = '' + maybeKey;
+    key = typeof maybeKey === 'symbol' ? maybeKey.toString() : '' + maybeKey;
```

Comment:
We shouldn't make the common case slower because of an extremely rare edge case that someone might pass a Symbol here. It's not worth the cost.